opstack/src/server/mod.rs

```diff
@@ -44,6 +61,7 @@ pub async fn start_server(
     let router = Router::new()
         .route("/latest", get(latest_handler))
         .route("/chain_id", get(chain_id_handler))
+        .route("/signer_proof/:block_number", get(signer_proof_handler))
```

Comment:
I think it would be safer if we pinned the proof to the block hash rather than the number. It's possible that the block at number n is different for the server and the light client (do to a fork).

Reply:
Makes sense